development/metamaskbot-build-announce.js

```diff
@@ -92,7 +85,10 @@ async function start() {
 
   const contentRows = [
     `builds: ${buildLinks}`,
-    `bundle viz: ${bundleMarkup}`,
+    `<details>
```

Comment:
I would have liked to clean up the spacing some by applying `margin-bottom: 0` here, but looks like GH strips attributes

Reply:
I moved the bundle viz links to the last row and this solves the aforementioned issue